exampleCourse/questions/element/multipleChoice/question.html

```diff
@@ -1,263 +1,297 @@
+      <pl-answer feedback="Incorrect. The sky is not brown.">Brown</pl-answer>
+      <pl-answer feedback="Incorrect. The sky is not red.">Red</pl-answer>
+    </pl-multiple-choice>
+  </div>
 </div>
 
 <div class="card my-2">
-    <div class="card-header">
-        Part 8
-    </div>
-
-    <div class="card-body">
-
-        <pl-question-panel>
-            <p>Here, you can observe the behavior of the <code>feedback</code> attribute in <code>pl-answer</code>. Feedback is expected to convey a small piece of information that explains why an
-                answer is correct or incorrect. When an answer with a feedback setting is selected and graded, a small box containing the feedback is displayed next to the answer.</p>
-
-            <p> What is the color of the sky? </p>
-        </pl-question-panel>
-
-        <pl-multiple-choice answers-name="sky-color8">
-            <pl-answer correct="true" feedback="Correct! The sky is indeed blue.">Blue</pl-answer>
-            <pl-answer feedback="Incorrect. The sky is not pink.">Pink</pl-answer>
-            <pl-answer feedback="Incorrect. The sky is not purple.">Purple</pl-answer>
-            <pl-answer feedback="Incorrect. The sky is not orange">Orange</pl-answer>
-            <pl-answer feedback="Incorrect. The sky is not yellow.">Yellow</pl-answer>
-            <pl-answer feedback="Incorrect. The sky is not brown.">Brown</pl-answer>
-            <pl-answer feedback="Incorrect. The sky is not red.">Red</pl-answer>
-          </pl-multiple-choice>
-
-    </div>
+  <div class="card-header">Part 9</div>
+
+  <div class="card-body">
+    <pl-question-panel>
+      <p>
+        Here, you can observe the behavior of the <code>score</code> attribute in
+        <code>pl-answer</code>. The score for a question the percentage of points given to it if
+        selected (default is 0 for incorrect answers and 1 for correct ones.)
+      </p>
+
+      <p>What is the color of the sky?</p>
+    </pl-question-panel>
+
+    <pl-multiple-choice answers-name="sky-color9">
+      <pl-answer correct="true" feedback="Correct! The sky is indeed blue.">Blue</pl-answer>
+      <pl-answer>Pink</pl-answer>
+      <pl-answer score="0.5">Purple</pl-answer>
+      <pl-answer feedback="Not quite" score="0.1">Orange</pl-answer>
+      <pl-answer>Yellow</pl-answer>
+      <pl-answer score="0.3">Brown</pl-answer>
+      <pl-answer score="0.2">Red</pl-answer>
+    </pl-multiple-choice>
+  </div>
 </div>
 
 <div class="card my-2">
-    <div class="card-header">
-        Part 9
-    </div>
-
-    <div class="card-body">
-
-        <pl-question-panel>
-            <p>Here, you can observe the behavior of the <code>score</code> attribute in <code>pl-answer</code>.
-            The score for a question the percentage of points given to it if selected (default is 0 for
-            incorrect answers and 1 for correct ones). This also allows no submission by the student, done by
-            setting <code>allow-blank="true"</code>.</p>
```

Comment:
Was `allow-blank` removed on purpose here?

Reply:
Whoops, yeah this was a mistake.

Reply:
Added back here: https://github.com/PrairieLearn/PrairieLearn/pull/8903/commits/1d6b1fd4e6a5e78c2c5377571ff130cdde3ad1c3